test/screenshot/infra/commands/test.js

```diff
@@ -462,8 +462,6 @@ ${CliColor.bold.red('Skipping screenshot tests.')}
    */
   logComparisonResults_(reportData) {
     this.logger_.foldStart('screenshot.diff_results', 'Diff results');
-    this.logComparisonResultSet_('Skipped', reportData.screenshots.skipped_screenshot_list);
```

Comment:
Is there a reason to remove Skipped? It shouldn't show up in practice on Travis but might be useful locally, IIUC...

Reply:
Good point. Reverted.